src/api/app/models/unregistered_user.rb

```diff
@@ -48,9 +48,11 @@ def self.register(opts)
     state = ::Configuration.registration == 'allow' ? "confirmed" : "unconfirmed"
 
     newuser = User.create(
-        login: opts[:login],
-        password: opts[:password],
-        email: opts[:email] )
+        login:                 opts[:login],
+        password:              opts[:password],
+        password_confirmation: opts[:password],
```

Comment:
From the documentation:

> If password confirmation validation is not needed, simply leave out the value for password_confirmation (i.e. don't provide a form field for it). When this attribute has a nil value, the validation will not be triggered.

http://api.rubyonrails.org/classes/ActiveModel/SecurePassword/ClassMethods.html

Reply:
Thanks